scrunch/datasets.py

```diff
@@ -1332,19 +1332,20 @@ def stream_rows(self, columns):
                                  {a: columns[a][x] for a in columns})
         return count
 
-    def push_rows(self, count):
+    def push_rows(self, count=None):
         """
         Batches in the rows that have been recently streamed. This forces
         the rows to appear in the dataset instead of waiting for crunch
         automatic batcher process.
         """
-        self.resource.batches.create({
-            'element': 'shoji:entity',
-            'body': {
-                'stream': count,
-                'type': 'ldjson'
-            }
-        })
+        if bool(self.resource.stream.body.pending_messages):
```

Comment:
We should provide a helper that creates the shoji wrappers. I see we're constructing shoji envelopes in many ways in different parts of the code. Something simple as `as_entity = lamda body: {"element": "shoji:entity", "body": body}` could do and give uniformity moving forward.